### DISCUSSION TITLE
Any way to use mobile_emulate?

### DISCUSSION THREAD

**Discussion by K4zuki-dev** [ultrafunkamsterdam/undetected-chromedriver#1065] [Q&A]
Quesiton is already in the title, is there any way to emulate an mobile phone with this? Just like Selenium (normal) the mobile_emualtion

**Comment by huynhit-biztex-92** [comment]
how about user-agent?
```py
options.add_option(--user-agent="Mozilla/5.0 (iPhone; CPU iPhone OS 10_3_1 like Mac OS X) AppleWebKit/603.1.30 (KHTML, like Gecko) Version/10.0 Mobile/14E304 Safari/602.1")
```

**Comment by ttlns** [comment]
As far I know, this currently isn't possible.
You could try using [Selenium-Profiles](https://github.com/kaliiiiiiiiii/Selenium-Profiles) tho.

As [issue](https://github.com/kaliiiiiiiiii/Selenium-Profiles/issues/26), it currently doesn't manage to be fully undetected by itsself, but wrapping it around undetected-chromedriver works as intended.

A example script would be:
```python
from selenium_profiles.driver import driver as mydriver
from selenium_profiles.profiles import profiles
from selenium.webdriver.common.by import By  # locate elements

mydriver = mydriver()
driver = mydriver.start(profiles.Android(), uc_driver=True)  # or .Windows()

# get url
driver.get('https://browserleaks.com/client-hints')  # test client hints

input("Press ENTER to exit: ")
driver.quit()  # Execute on the End!
```
